content/blog/2023-01-04-alert-migrate-your-etc-endpoints-from-ethercluster-to-rivet-by-july-1st-2023/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Alert: Migrate Your ETC Endpoints From Ethercluster to Rivet by July 1st, 2023"
+date: 2023-01-04
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["News", "Announcement"]
+linkImage: ./etc-endpoint-alert-banner.png
+---
+
+---
+**Watch the ETC Cooperative's announcement here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/gYaL-yJCPB0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+## IMPORTANT ALERT:
+
+The [ETC Cooperative has announced](https://etccooperative.org/posts/2023-01-02-the-ethereum-classic-rpc-url-is-changing-from-ethercluster-to-rivet-en) that by July 1st of 2023 it will discontinue the RPC endpoint for Ethereum Classic under the Ethercluster brand and will fully migrate the service to Rivet, a blockchain cloud service.
+
+![From Ethercluster to Rivet.](./etc-endpoint-alert-banner.png)
+
+The Ethercluster endpoint that will be discontinued by July 1st of 2023 is:
+
+[https://ethercluster.com/etc](https://ethercluster.com/etc)
+
+The migration may be done immediately to the following RPC endpoint:
+
+[https://etc.rivet.link](https://etc.rivet.link)
+
+If you are a wallet service, node operator, mining pool, dapp developer, or any other kind of user of the Ethercluster endpoints, please migrate as soon as possible to avoid interruptions.
+
+The last day for the current Ethercluster rpc endpoint is July 1st of 2023.
+
+## Summary
```

Comment:
I think you can omit the Summary section at the end.  It is redundant, especially with the article now being so short.
Just keep the WARNING AT THE END, but not the Summary heading or body.

Also worth saying that alternative endpoints are also available.   See /network/endpoints/ for more details.